### PR TITLE
Describe how to configure automatic image signature import

### PR DESCRIPTION
**Pull request by mfojtik**
Describe how to configure: https://github.com/openshift/origin/pull/16293

### CODE REVIEW

**Comment by mfojtik** [comment]
@mtrmac @runcom do you guys have some github pages or official web site we can link for documentation? Not sure if linking to a github markdown file is good idea long term ;-)

**Comment by mtrmac** [comment]
If you are asking about https://github.com/containers/image/blob/master/docs/registries.d.md, that’s it; I don’t think we have a more fancy website. I guess you could also link to more guide-like articles at access.redhat.com.

File: admin_guide/image_signatures.adoc [hunk 1]
```diff
@@ -247,3 +247,25 @@ the `<name>` is the name of the signature. The signature name must be 32
...
+automatically for all Red Hat images:
+
+----
+docker:
+  registry.access.redhat.com:
+    sigstore: https://access.redhat.com/webassets/docker/content/sigstore
+----
+
+Note that all signatures imported automatically by {{product-title}} will be
+"unverified" by default and will have to be verified by image administrators.
```
Comment (thomasmckay) [inline admin_guide/image_signatures.adoc]: Instructions for verification command and link to signature docs would be appreciated.

**Comment by adellape** [comment]
[Document: Import signatures from upstream registry images](https://trello.com/c/Fr9NkSKW/695-document-import-signatures-from-upstream-registry-images)

**Comment by mburke5678** [comment]
@mfojtik What is the status of this PR? I have been asked to take over the associated Trello from Alex D.

**Comment by mfojtik** [comment]
@thomasmckay FYI

**Comment by mburke5678** [comment]
@mfojtik @thomasmckay I need to get this card to QE by EOB Wednesday. Please let me know if this doc is technically complete by then so I can send it to QE.

**Comment by mburke5678** [comment]
@mfojtik @zhouying7780 has a question on the need to restart services after editing the YAML. If you agree that this is needed, please make the changes or let me know and I can merge what you have and do a follow-up PR to add the restart. Would like to close on this.

**Comment by mburke5678** [comment]
@mfojtik @zhouying7780 @thomasmckay Please see question above on restart. I need to close by end-of-week. If I do not hear back, I will add the restart step and move this issue along. We can remove later, if needed.

**Comment by mfojtik** [comment]
@mburke5678 the restart is not required, you just need to make that file accessible by master. If you run the master in container you might restart if you bind mounting the configuration file.

**Comment by mburke5678** [comment]
@mfojtik  @zhouying7780  @thomasmckay 
How does this look? [Follow-up edit to Describe how to configure automatic image signature import](https://github.com/openshift/openshift-docs/pull/6381/files) 

I brought down some information from the github file and moved the link to the end to de-emphasize it.

**Pull request merged**